### ISSUE TITLE
0.13: [Bug]: Logic for parsing Dockerfiles and computing inclusions is missing for action configs

### ISSUE THREAD

**Issue by vvagaytsev** [garden-io/garden#5523]
The logic for parsing Dockerfiles and computing the file inclusions from them is only being used for module configs, not action configs. That needs to be fixed.

### Current Behavior

Dockerfiles are parsed and processed only when using module configs.

### Expected behavior

Dockerfiles should be parsed and processed when using both action and module configs.

### Workaround

None.

### Additional context

See `containerHelpers.autoResolveIncludes` helper.

### Your environment

* **OS:** 
* **How I'm running Kubernetes:**  

`garden version 0.13.21`

**Comment by thsig** [comment]
I seem to recall this being fixed the other day—do you remember, @vvagaytsev?

**Comment by vvagaytsev** [comment]
@thsig yes, we still need to fix it. I haven't had a chance to do it so far.
